src/components/sds/Typography/index.tsx

```diff
@@ -309,15 +315,29 @@ export const Text: React.FC<TextProps> = ({
   color,
   children,
   isVerticallyCentered = false,
+  url,
   ...props
-}) => (
-  <StyledText
-    $size={getSize({ size, ...props }, "md")}
-    $weight={getWeight({ weight, ...props }, "regular")}
-    $color={getColor({ color, ...props }, THEME.colors.text.primary)}
-    $isVerticallyCentered={isVerticallyCentered}
-    {...props}
-  >
-    {children}
-  </StyledText>
-);
+}) => {
+  const handlePress = () => {
+    if (url) {
+      Linking.canOpenURL(url).then((supported) => {
+        if (supported) {
+          Linking.openURL(url);
+        }
+      });
```

Comment:
Since we don't have a tracking error tool installed it (like Sentry), could you please add a TODO here for us to catch and track the error once we have a tracking error mechanism in place? Wdyt?